packages/replay/metadata/test.js

```diff
@@ -0,0 +1,43 @@
+const {
+  array,
+  create,
+  defaulted,
+  enums,
+  number,
+  object,
+  optional,
+  string,
+} = require("superstruct");
+
+const VERSION = 1;
+
+const versions = {
+  1: object({
+    version: defaulted(number(), () => 1),
+    title: string(),
+    result: enums(["passed", "failed", "timedOut"]),
+    path: optional(array(string())),
+    run: optional(string()),
+    file: optional(string()),
```

Comment:
This is a place where I actually think sorted keys might be nice, but I don' feel strongly about it.